MediaBrowser.MediaEncoding/Encoder/EncoderValidator.cs

```diff
@@ -130,6 +115,7 @@ private enum Codec
             Decoder
         }
 
+        // When changing this, also change the minimum library versions in _ffmpegMinimumLibraryVersions
         public static Version MinVersion { get; } = new Version(4, 0);
 
```

Comment:
Since we're shipping with FFmpeg v4.3 due to some hw acceleration requirements it would probably make sense to update the MinVersion to 4, 3?

Reply:
I would perfer 4.2. see https://github.com/jellyfin/jellyfin/pull/3575#discussion_r456948670